game.js

```diff
@@ -56,3 +55,24 @@ function playRound(playerSelection, computerSelection) {
     roundScore.textContent = "It's a tie!";
   }
 }
+function game() {
+  while (computerScore !== 5 || playerScore !== 5) {
+    const buttons = document.querySelector(".js-buttons-container");
+    buttons.addEventListener("click", choosePlayer);
```

Comment:
You are adding `choosePlayer` event listeners in a loop. There is no need in loops at all.
- Add event listeners only once.
- When the user clicks the button:
  - Play computer.
  - Compare with the player's choice.
  - Display the results.
  - Check the score.
    - If `5`
      - Display the end of the game.
      - Disable the buttons so that the game could not continue.
    - If less than `5`.
      - Do nothing. The player will be able to continue clicking the buttons.